File: INFO1910/transpiler/test/functions/add_2.c
#include <stdio.h>

int add_2(int x);

int main(){
    int x = 5;
    x = add_2(x);
    printf("%d\n", x);
    return 0;
}

int add_2(int x){
    x = x + 2;
    return x;
}

File: Subject Archive/INFO1910 Intro to programming/transpiler/test/testcase/add_2.py
import sys

def main(argc, argv):

    x = int( 5)

    x = add_2(x);
    print("%d\n"%( x), end = "")
    return 0
    
def add_2(x):

    x = x + 2
    return x
    
CALLMAINFUNCTIONVAR = main(len(sys.argv), sys.argv)
